### ISSUE TITLE
loss is nan at the begining

### ISSUE THREAD

**Issue by ustczhouyu** [facebookresearch/maskrcnn-benchmark#1211]
## ❓ Questions and Help
help!! when I train my own dataset, the loss is nan at the begining, can anybody tell me how to deal with it? thanks a lot!!

![image](https://user-images.githubusercontent.com/38032284/72412139-9b382400-373a-11ea-96b9-f1b083b21637.png)

**Comment by civilpat** [comment]
Hope you have solved the issue. If not, you may either lower your base learning rate or double check your own dataset. My experience is that sometimes incorrect annotations may also cause the issue.
